src/pages/authors/authorsCreate.vue

```diff
@@ -16,11 +16,38 @@
 </template>
 
 <script>
+import { mapGetters, mapActions } from 'vuex'
+
 export default {
   data () {
     return {
-      name: '',
-      email: ''
+      value: {
+        name: '',
+        email: ''
+      }
+    }
+  },
+
+  computed: {
+    ...mapGetters({
+      authors: 'authors/authorsList'
+    })
+  },
+
+  created () {
+    this.fetchAuthors()
+  },
+
+  methods: {
+    ...mapActions({
+      fetchAuthors: 'authors/fetchAuthors'
+    }),
+
+    saveAuthor () {
+      this.$axios.post('authors', this.value)
```

Comment:
naõ entendi mt bem o que vc ta fazendo nesse trecho, pode me explicar?

Reply:
não deveria ser uma action?

Reply:
Luis, faz a mesma coisa que > saveAuthor () {
      this.authorsCreate(this.values)
    }
o authorsCreate é uma action que faz o post > api.post('authors', values)

Elias, já mudei para action